agent/consul/status_endpoint_test.go

```diff
@@ -24,6 +25,23 @@ func rpcClient(t *testing.T, s *Server) rpc.ClientCodec {
 	return msgpackrpc.NewClientCodec(conn)
 }
 
+func insecureRPCClient(s *Server, c tlsutil.Config) (rpc.ClientCodec, error) {
+	addr := s.config.RPCAdvertise
+	configurator, err := tlsutil.NewConfigurator(c, nil)
+	if err != nil {
+		return nil, err
+	}
+	wrap := configurator.OutgoingRPCWrapper()
```

Comment:
Renaming this to `wrapper` might be a little clearer, since wrapper is also used in `DialTimeoutInsecure`

Reply:
✔️